src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php

```diff
@@ -159,11 +159,34 @@ private function addUser(string $userId, string $name, string $password): void
             'show_as_rights_template' => '1',
         ]);
 
-        $this->databaseConnection->insert('cms_user_cms_language_mlt', [
-            'source_id' => $userId,
-            'target_id' => $languageEn->id,
-            'entry_sort' => 1,
-        ]);
+        $this->addLanguageLinks($userId);
```

Comment:
connectUserWithPortals? (or similar) "Links" is a bit arbitrary

Reply:
If it makes you happy .. ;-)